2주차/1920.py

```diff
@@ -0,0 +1,11 @@
+# 1920_수 찾기
+import sys
+
+n=int(sys.stdin.readline())
+nn=set(map(int, sys.stdin.readline().strip().split()))
+
+m=int(sys.stdin.readline())
+mm=list(map(int, sys.stdin.readline().strip().split()))
+
+for num in mm:
```

Comment:
```suggestion
mm=map(int, sys.stdin.readline().strip().split())

for num in mm:
```
for문에 한 번 쓰고 말거라면 굳이 list로 안바꾸고 map자체도 for문에 쓸 수 있음
대신 map은 일회용이라 그 다음에 다시 리스트로 바꾼다거나 for문에 쓰려고 하면 아무것도 안나옵니다